### DISCUSSION TITLE
Letterboxd not showing any ratings.

### DISCUSSION THREAD

**Discussion by DROIDshield** [jurialmunkey/skin.arctic.horizon.2#892] [Discussion]
Been looking into letterboxd and was hoping the ratings might populate.  Done some online reading and it seems this runs off TMDB api,? 

What do I need to do to get it was everything I try it doesn't show anything.

**Comment by xyzfre** [comment]
Did you put your mdblist API key in TMDb settings this is what I did and the next day they showed up for all movies on my end they took about 24 hours to show up

**Reply by DROIDshield** [reply]
I've just generated an mdblist api as it's not one I had before so will put it in TMDB shortly. Thanks for the help. Hopefully this will work.

**Reply by DROIDshield** [reply]
Added key, working like a treat. Again thanks for the info and @jurialmunkey your work is sublime...  thanks again.  I will be going over to the donate page... oh and thanks for popping my name in the thanks page...top man. 
![20230401_172503](https://user-images.githubusercontent.com/97023369/229302843-1df7d32e-c0aa-40e2-998d-4093cd28db2a.jpg)

**Reply by xyzfre** [reply]
You also get extra ratings now for TV shows not sure if you're aware you just have to enable them in AH2 settings,Ratings ,,loving this new feature as well I like seeing Ratings everywhere lol

**Reply by DROIDshield** [reply]
Yes I am aware of those... thing's this skin absolutely wipes the floor with TBM I used to use much lighter and the updates and fixes are rapid.
